### ISSUE TITLE
Is it possible to automatically populate 'Actor name' or 'they' in step description?

### ISSUE THREAD

**Issue by nbarrett** [serenity-bdd/serenity-core#777]
I prefer to use quite specific Actor names that include such things as their role(s) as the system we are testing is extremely role-specific. Using James or Joe is provides an unnecessary level of indirection for me in my particular case. The trouble with this  is that for 'built-in' screenplay tasks the html report looks rather cluttered e.g.

![image](https://cloud.githubusercontent.com/assets/626941/25657033/b4fc9822-2ff3-11e7-989b-45837d10502e.png)

(Note in the above, the last task is one of mine so I just use 'They' in the task description)

Is there a way to choose whether to insert the **Actor name** or **They** depending on whether the task is a top level one or not? In practice this makes sense as at the top level you talk about the actor, and then in the drill-down, you say what **they** are up to. Currently I have a mixture of this:

```
@Step("{0} double-clicks on #target")
```

and this:

```
@Step("They double-click on #target")
```

But it would be nice for the population to be context-sensitive. Can this be done?

**Comment by wakaleo** [comment]
That wouldn't be hard to do in a boolean sort of way (i.e. for all tests or for none). It would be more complicated to make it more contextual.

**Comment by nbarrett** [comment]
Apart from the complexity of working out whether the current task is a top level one or not, there is another gotcha! The English language dictates that the verb is plural when the actor (singular) is referred to and the verb is singular when they (singular also) is referred to :smile: Might I say that complexity has 
never stopped you from overcoming a challenge before..?!

**Comment by wakaleo** [comment]
The language issue is a good point - one minute you are going from 3rd person plural to 3rd person singular, the next you are declining Polish verbs ;-).

**Comment by wakaleo** [comment]
Fixed/implemented in 1.9.0

**Comment by nbarrett** [comment]
Thanks @wakaleo - I'll take a look in 1.9!
